gallery_dl/extractor/tcbscans.py

```diff
@@ -0,0 +1,107 @@
+    test = (
+        (("https://onepiecechapters.com"
+          "/chapters/4708/chainsaw-man-chapter-108"), {
+            "pattern": (r"https://cdn\.[^/]+"
+                        r"/(file|attachments/[^/]+)/[^/]+/[^.]+\.\w+"),
+            "count": 17,
+            "keyword": {
+                "manga": "ChainSaw Man",
+                "chapter": 108,
+                "chapter_minor": "",
+                "lang": "en",
+                "language": "English",
+            }
+        }),
+        ("https://onepiecechapters.com/chapters/4716/one-piece-chapter-1065", {
+            "pattern": (r"https://cdn\.[^/]+"
+                        r"/(file|attachments/[^/]+)/[^/]+/[^.]+\.\w+"),
+            "count": 18,
+            "keyword": {
+                "manga": "One Piece",
+                "chapter": 1065,
+                "chapter_minor": "",
+                "lang": "en",
+                "language": "English",
+            }
+        }),
+        (("https://onepiecechapters.com/"
+          "chapters/44/ace-novel-manga-adaptation-chapter-1")),
+    )
+
+    def images(self, page):
+        return [
+            (url, None)
+            for url in text.extract_iter(
+                page, '<img class="fixed-ratio-content" src="', '"')
+        ]
+
+    def metadata(self, page):
+        manga, _, chapter = text.extract(
+            page, 'font-bold mt-8">', "</h1>")[0].rpartition(" - ")
+        chapter, sep, minor = chapter[8:].partition(".")
+        return {
+            "manga": text.unescape(manga),
+            "chapter": text.parse_int(chapter),
+            "chapter_minor": sep + minor,
+            "lang": "en", "language": "English",
+        }
+
+
+class TcbscansMangaExtractor(TcbscansBase, MangaExtractor):
+    chapterclass = TcbscansChapterExtractor
+    pattern = (r"(?:https?://)?onepiecechapters\.com"
+               r"(/mangas/\d+/[^/?#]+)")
+    test = (
+        ("https://onepiecechapters.com/mangas/13/chainsaw-man", {
+            "pattern": TcbscansChapterExtractor.pattern,
+            "count": ">= 110",
+        }),
+        ("https://onepiecechapters.com/mangas/4/jujutsu-kaisen", {
+            "pattern": TcbscansChapterExtractor.pattern,
+            "count": ">= 205",
+        }),
+        ("https://onepiecechapters.com/mangas/15/hunter-x-hunter"),
+    )
+
+    def chapters(self, page):
+        data = {
+            "manga": text.unescape(text.extr(
+                page, 'class="my-3 font-bold text-3xl">', "</h1>")),
+            "description": text.unescape(text.extr(
+                page, 'class="leading-6 my-3">', "</p>")),
+            "lang": "en", "language": "English",
+        }
+
+        results = []
+        page = text.extr(page, 'class="col-span-2"', 'class="order-1')
+        for chapter in text.extract_iter(page, "<a", "</a>"):
+            url = text.extr(chapter, 'href="', '"')
+            data["title"] = text.unescape(text.extr(
+                chapter, 'text-gray-500">', "</div>"))
```

Comment:
I don't know if I should extract the 'title', since they're available on the manga page, but not on individual chapters